docs/sources/tasks/collect-logs-in-kubernetes.md

```diff
@@ -0,0 +1,323 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/tasks/collect-logs-in-kubernetes/
+description: Learn how to collect and forward logs from Kubernetes to Loki
+title: Collect and forward Kubernetes logs
+weight: 200
+---
+
+# Collect logs and forward Kubernetes logs
+
+You can configure {{< param "PRODUCT_NAME" >}} to collect logs and forward them to a [Loki][] database.
+
+This topic describes how to:
+
+* Configure logs delivery.
+* Collect logs from Kubernetes Pods.
+
+## Components used in this topic
+
+* [discovery.kubernetes]
+* [discovery.relabel]
+* [local.file_match]
+* [loki.source.file]
+* [loki.source.kubernetes]
+* [loki.source.kubernetes_events]
+* [loki.process]
+* [loki.write]
```

Comment:
```suggestion
* [discovery.kubernetes][]
* [discovery.relabel][]
* [local.file_match][]
* [loki.source.file][]
* [loki.source.kubernetes][]
* [loki.source.kubernetes_events][]
* [loki.process][]
* [loki.write][]
```

Not a hard requirement, but suggesting the change for consistency with other topic syntax/style.